Formula/gitlab-ci-multi-runner.rb

```diff
@@ -75,6 +75,36 @@ def install
     end
   end
 
+  plist_options :startup => true, :manual => "gitlab-ci-multi-runner start"
+
+  def plist; <<-EOS.undent
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>SessionCreate</key><false/>
+        <key>KeepAlive</key><true/>
+        <key>RunAtLoad</key><true/>
+        <key>Disabled</key><false/>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/gitlab-ci-multi-runner</string>
+          <string>run</string>
+          <string>--working-directory</string>
+          <string>#{File.expand_path("~")}</string>
+          <string>--config</string>
+          <string>#{File.expand_path("~")}/.gitlab-runner/config.toml</string>
```

Comment:
We'll need to figure out a way to make it clearer that this requires configuration before the plist can be used. To me that makes it a bad plist candidate, personally.